Lesson32/Kadir/my-app/src/components/RecipeList/index.jsx

```diff
@@ -1,13 +1,39 @@
-export const RecipeList = ({recipes}) => {
+import "./styles.css";
+import { useState } from "react";
 
-    return(
-        <div>
-            <h2>Search Results</h2>
-            <ul>
-              {
-             recipes.map(recipe => <li key={recipe.idMeal} >{recipe.strMeal}</li>)   
+export const RecipeList = ({ recipes, handleRecipeSelect, selectedRecipe }) => {
+  const [isActive, setIsActive] = useState(true);
```

Comment:
If you do changes as suggested below, you can remove the following:
1. `import { useState } from "react";`
2. `const [isActive, setIsActive] = useState(true);`
3. ` const toggleClick = () => {
    setIsActive(!isActive);
  };`